timemachine/cpp/src/context.cu

```diff
@@ -37,6 +39,8 @@ Context::Context(
     cub::DeviceReduce::Sum(d_sum_storage_, d_sum_storage_bytes_, d_in_tmp, d_out_tmp, N_);
     gpuErrchk(cudaPeekAtLastError());
     cudaSafeMalloc(&d_sum_storage_, d_sum_storage_bytes_);
+
+    get_nonbonded_all_pair_potentials(bps, nonbonded_pots_);
```

Comment:
(maybe add a note that this is a no-op if we're running vacuum or if the nonbonded all pairs are missing).

Reply:
Added check in https://github.com/proteneer/timemachine/pull/1053/commits/968c2a8d954cd3e3a851a6703bcc64ee46543f04